### ISSUE TITLE
[MOTION/COGNITION]: Auto-scrolling problematic for users with motion sensitivity or cognitive considerations

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#6461]
## Description

The wizard on the [Discharge Upgrade page](https://www.va.gov/discharge-upgrade-instructions) automatically scrolls users around the form when a question is answered or a response is changed. This also occurs if you click the `edit` link on the review screen.

This automatic scrolling can cause issues for users with motion sensitivity or cognitive considerations.

## Point of Contact

Liz Lantz 

## Acceptance Criteria

I'm not sure exactly what the right answer is here for AC.  I think that the animation should be removed from the form fields so the user can control their own destiny, so to speak.  I expected the `edit` link to open an accordion or similar where I could edit my answer right there.  I default to our 508 experts to make recommendations on best practices there.

## Environment
Macbook, chrome

## Steps to Recreate

- Visit the [Discharge Upgrade page](https://www.va.gov/discharge-upgrade-instructions)
- Go through the wizard
- try changing a response to a previously answered question before you get to the review step
- on the review step, edit one of your responses

**Comment by andaleliz** [comment]
Hey @joshkimux, @johnhashva and @MarciMcGuireGCIO! I opened this issue a while back and it looks like it never got an assignee.  I believe this falls into the decision tools portfolio, but if another team is responsible, please feel free to re-assign.

Josh, including you on this in case you disagree with me that it is a problem and it can be closed with no action.

**Comment by MarciMcGuireGCIO** [comment]
Hey @Liz, I will find out who this should get assigned to.  Currently we are managing Find a VA Form and Find a Yellow Ribbon School.

**Comment by johnhashva** [comment]
@MarciMcGuireGCIO just a reminder: you should check with @leahkeeler and @vhenry07 to confirm this tool fits in one of their portfolio/suites (that was the guidance I received from @mattpointzxer0 few days ago).

**Comment by bkjohnson** [comment]
Actually, the solution on the new KMI tool was using the new `<va-accordion>` instead of the older `<CollapsiblePanel>` React component which has this problematic behavior. [Documentation available here](https://dev-design.va.gov/storybook/?path=/docs/components-va-accordion--default)

**Comment by k80bowman** [comment]
Need to validate if this is still an issue.

**Comment by MarciMcGuireGCIO** [comment]
We have made no changes to address this issue.

**Comment by CourtneyChambers77** [comment]
A switch control that can toggle on/off the infinite scroll even before interacting with infinite scroll content. Ideally, the switch control can be placed before the role=feed and can be made visible when it receives keyboard focus.
Load more or pagination links must appear once the infinite scroll is switched-off so that user has the luxury of navigating the content at leisure.

**Comment by MarciMcGuireGCIO** [comment]
Looping in new PM Nicole Leonard.

**Comment by joshkimux** [comment]
@MarciMcGuireGCIO I just checked this out and this is still an issue. Ideally we'd like to get this fixed using the guidance in the [comments above which removes the autoscroll animation](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6461#issuecomment-811093499) 🙏 

This would be a `defect-0` as it presents repetitive potential dangers towards Veterans with vestibular disorders. Has the discharge wizard ever received an official staging review? There are a number of other a11y issues that need to be addressed-- notably a lack of a continue button, skipped headings (jumps from 1 to 4), and a lack of `aria` for content inbetween interactive elements in a form.

@CourtneyChambers77 , for some context just in case, this issue is addressing the motion related issues when navigating through the form (see video below). We can update the ticket in the future to be a little bit more descriptive as I know `auto-scrolling` can mean several things 🙇  

https://user-images.githubusercontent.com/14154792/146295304-dd280992-94e5-4c69-9f24-07f9f6d76976.mov

**Comment by k80bowman** [comment]
@MarciMcGuireGCIO It sounds like this is something that is not the design system team's responsibility to fix, is that correct?

**Comment by MarciMcGuireGCIO** [comment]
@k80bowman I can't really say for certain.  A FE engineer would probably need to look at the code to see if it utilizes components from the design system.  I suspect not, but my FE engineer made only minor text updates to it and brought some of the testing up to standards.

**Comment by k80bowman** [comment]
@MarciMcGuireGCIO I took a quick look at the code to see what the problem might be and it looks like there are some scrolling functions in [this file](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/discharge-wizard/components/FormQuestions.jsx) that are causing the issue (particularly the [scrollToLast](https://github.com/department-of-veterans-affairs/vets-website/blob/17f5fd3b17c0402739d912fc180f88a6f0bce41b/src/applications/discharge-wizard/components/FormQuestions.jsx#L27) function). 

Since this is not a design system team problem, I'm going to remove our labels from this card.

**Comment by taylorkaren** [comment]
Removing the forms label from this issue as it is not a Forms Library fix.

**Comment by joshkimux** [comment]
This remains to be an open, critical accessibility issue. @coforma-terry is there any way we can loop in CAIA to support?

**Comment by joshkimux** [comment]
@MarciMcGuireGCIO is @nicole-leonard still the PM responsible for resolving this ticket?

**Comment by laflannery** [comment]
Closing as this as been resolved with the launch of the redesigned DUW using the subtask pattern @FranECross @artsymartha68
